apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/RealSubscriptionManager.java

```diff
@@ -131,7 +166,7 @@ void doUnsubscribe(Subscription subscription) {
     SubscriptionRecord subscriptionRecord;
     synchronized (this) {
       subscriptionRecord = subscriptions.remove(subscriptionId);
-      if (subscriptionRecord != null && state == State.ACTIVE) {
+      if ((subscriptionRecord != null) && (state == State.ACTIVE || state == State.STOPPING)) {
         transport.send(new OperationClientMessage.Stop(subscriptionId));
       }
 
```

Comment:
I think that we shouldn't call `startInactivityTimer ` in case of `state == State.STOPPING`. Doesn't make any sense right?